### PR TITLE
Replacing user-group with community forum, added link to U. lesson on…

### PR DESCRIPTION
**Pull request by guy9**
… Spring Boot
@annastuchlik please review

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Asking questions or requesting help
 
-Use the [Scylla Users mailing list](https://groups.google.com/g/scylladb-users) or the [Slack workspace](http://slack.scylladb.com) for general questions and help.
+Use the [ScyllaDB Community Forum](https://forum.scylladb.com) or the [Slack workspace](http://slack.scylladb.com) for general questions and help.
```
Comment (nyh) [inline CONTRIBUTING.md:5]: Why replace instead of add?
Have we made an official decision to close the scylladb-users mailing list?

Comment (tzach) [reply CONTRIBUTING.md:5]: We are not closing the mailing list, but the focus is the forum, and we prefer users to go there with questions (and answers)

**Review by annastuchlik — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (guy9) [reply CONTRIBUTING.md:5]: @nyh, the mailing list won't be closed at this point, but it won't be promoted either. We have already started moving announcements to the community forum and only posting a link to the thread from the mailing list.

Comment (nyh) [reply CONTRIBUTING.md:5]: @avikivity if you agree with this direction, I'll you have the honor of merging it.

Comment (guy9) [reply CONTRIBUTING.md:5]: @avikivity please have a look

**Comment by avikivity** [comment]
Please fix your name and email address in the patch metadata.

File: CONTRIBUTING.md (hunk 1 above)
Comment (guy9) [reply CONTRIBUTING.md:5]: Thanks @avikivity , I fixed the metadata

**Pull request closed without merging**